### PR TITLE
Include release note examples in PR template.

### PR DESCRIPTION
**Pull request by moorepants**
#### Brief description of what is fixed or changed

I always forget the format and it's a bit annoying to have to navigate to the wiki to read the docs there. It'd be nice if the format of the release notes are included in the PR template.

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by oscarbenjamin** [comment]
Looks good

**Comment by oscarbenjamin** [comment]
Tests have failed due to #20946

**Comment by JSS95** [comment]
Is there any more issue blocking this to be merged?

**Comment by moorepants** [comment]
I think it is good to go. Merging.

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -17,11 +17,24 @@ open. -->
...
+
+* solvers
+  * Added a new solver for logarithmic equations.
+
+* functions
+  * Fixed a bug with log of integers.
+
+or if no release note(s) should be included use:
+
+NO ENTRY
```
Comment (asmeurer) [inline PULL_REQUEST_TEMPLATE.md:32]: IMO we should omit the `NO ENTRY` instructions. Too many people already skip writing release notes with `NO ENTRY` when they really should include one.

Comment (oscarbenjamin) [reply PULL_REQUEST_TEMPLATE.md:32]: The explanation of how to not include an entry needs to be somewhere. I now know it automatically but I used to have to check this instruction every time. The particular invocation "NO ENTRY" doesn't seem intuitive to me.

Comment (asmeurer) [reply PULL_REQUEST_TEMPLATE.md:32]: I always remember it, but I guess I also invented it :) Suggestions on a better way to write it are welcome.

It is still described on the wiki page which is linked to in the text here.

**Pull request merged**